src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,23 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String blade;
+
     public Bulldozer() {
+        super();
+    }
+
+    public Bulldozer(String blade) {
+        super();
+        this.blade = blade;
+    }
```

Comment:
Almost correct, but you don't need to call anything inside the empty construtor and in the second constructor you should also pass parameters that Machine has, then call `super()` with those params, so it would look like this:
```
public Bulldozer(String name, String color, String blade) {
        super(name, color);
        this.blade = blade;
    }
```
Same for Excavator and Truck.